src/methods/users.js

```diff
@@ -11,12 +11,19 @@ export default function users(): Object {
       });
     },
 
-    photos: (username: string, page: number = 1, perPage: number = 10, orderBy: string = "latest") => {
+    photos: (
+      username: string,
+      page: number = 1,
+      perPage: number = 10,
+      orderBy: string = "latest",
+      stats: bool = false,
+    ) => {
```

Comment:
I wonder if it would be useful to use named parameters here, now it's getting longer?